tests/unit/core/core.html

```diff
@@ -108,6 +108,79 @@
 
 <div id="dimensions" style="float: left; height: 50px; width: 100px; margin: 1px 12px 11px 2px; border-style: solid; border-width: 3px 14px 13px 4px; padding: 5px 16px 15px 6px;"></div>
 
+<div id="labels-fragment">
+	<label for="test">1</label>
+	<div>
+		<div>
+			<form>
+				<label for="test">2</label>
+				<label>3
+					<input id="test">
+				</label>
+				<label for="test">4</label>
+			</form>
+			<label for="test">5</label>
+		</div>
+		<div>
+			<div>
+				<form>
+					<label for="test">6</label>
+				</form>
+			</div>
+		</div>
+	</div>
+	<div>
+		<div>
+			<form>
+				<label for="test">7</label>
+				<label>
+				</label>
+				<label for="test">8</label>
+			</form>
+			<label for="test">9</label>
+		</div>
+		<div>
+			<div>
+				<form>
+					<input id="test-2">
+					<label for="test">10</label>
+				</form>
+			</div>
+		</div>
+	</div>
+</div>
+
+<div id="weird-['x']-id"></div>
+</div>
```

Comment:
if you mean that these are not in the test fixture, No. Though i did mean to add a comment to explain why.
If you place these in the test fixture they get reset / replaced between each async test which is not what i want. because then any saved references are to a detached fragment, not attached DOM which i specifically want to test.